### DISCUSSION TITLE
能否增加ipv6 nat即kmod-ipt-nat6 支持

### DISCUSSION THREAD

**Discussion by aieofwang** [coolsnowwolf/lede#9567] [R68S / 电犀牛专区]
# 希望固件增加nat6支持
学校的校园网会dhcp分发ipv6地址，但没有pd前缀
试过桥接，但过一阵就会失效，内网能拿到v6地址但无法上网
无奈使用nat6

**Comment by mistershxxx** [comment]
有同样的问题，在N1上可以使用，在r68s上ip6tables无法转发，重启防火墙出现：
ip6tables v1.8.3 (legacy): can't initialize ip6tables table `nat': Table does not exist (do you need to insmod?)
 Perhaps ip6tables or your kernel needs to be upgraded.
安装luci-proto-ipv6也没有dhcpv6选项
